packages/serverpod/test/database/columns/column_enum_test.dart

```diff
@@ -8,80 +8,159 @@ enum TestEnum {
 }
 
 void main() {
-  group('Given a ColumnEnum', () {
+  group('Given a ColumnEnumSerializedAsString', () {
```

Comment:
Chore: These tests should be separated into two test files named after the enum they are testing.